MANIFEST-packages.in

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+include NOTICE
+include LICENSE
+include CHANGELOG.txt
+include README.md
+graft licenses/
```

Comment:
This isn't working. The logs are showing:

> warning: Failed to find the configured license file 'licenses/*'

Reply:
Strange. It works for me.

Reply:
This was from CI logs

Reply:
This is warning is still showing up on CI @potiuk 


```
-----------------------------------------------------------------------------------

 Preparing backporting package mongo

-----------------------------------------------------------------------------------

warning: Failed to find the configured license file 'licenses/*'

warning: no previously-included files matching '__pycache__' found anywhere in distribution

warning: no previously-included files matching '*.pyc' found anywhere in distribution

warning: check: missing required meta-data: url

warning: build_py: byte-compiling is disabled, skipping.

warning: install_lib: byte-compiling is disabled, skipping.

gitpython not found: Cannot compute the git version.

running clean

removing 'build/lib' (and everything under it)

removing 'build/bdist.linux-x86_64' (and everything under it)

'build/scripts-3.6' does not exist -- can't clean it

removing 'build'
```